src/main/java/no/difi/sdp/client2/asice/CreateASiCE.java

```diff
@@ -79,6 +78,13 @@ private void writeArchiveToDisk(Archive archive) {
         }
     }
 
+
+    protected static long getUnzippedContentBytesCount(List<AsicEAttachable> attachables){
```

Comment:
Kombinasjonen `protected` og `static` er det skjelden jeg ser. Hva er bakgrunnen for det? Aksess i tester eller noe sånt?